hmm-lib/pom.xml

```diff
@@ -64,8 +64,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
```

Comment:
Can you just remove the entire plugin, please? It will be taken from parent pom then.